amy/workshops/views.py

```diff
@@ -1293,7 +1293,7 @@ def event_import(request):
         return HttpResponseBadRequest("Network connection error.")
 
     except WrongWorkshopURL as e:
-        return HttpResponseBadRequest(str(e))
+        return HttpResponseBadRequest(f"URL error: {e.msg}")
```

Comment:
Could we leave a note here in the comments mentioning why this was done?

Reply:
I added the comment to the `WrongWorkshopURL` class.